src/utils.jl

```diff
@@ -191,6 +191,24 @@ getstream(fmt, io::IOStream) = getstream(fmt, io, extract_filename(io))
 # assume OMETIFF if no format given
 getstream(io) = getstream(format"TIFF", io)
 
+@static if Sys.iswindows()
+    # Be permissive on windows with eager GC to work around
+    # https://github.com/tlnagy/TiffImages.jl/pull/79#discussion_r880478304
+    function _safe_open(f, filepath::String, mode="r", args...; kwargs...)
+        GC.gc()
+        try
+            open(f, filepath, mode, args...; kwargs...)
+        catch err
+            if err isa SystemError
+                @warn "failed to open file \"$filepath\" in \"$mode\" mode, this may be caused by overwriting a file previously opened with mmap"
```

Comment:
Should we advise them on a solution? Maybe "Try picking a new file name or restarting your Julia session."?

Reply:
I'm not sure -- the solution to it depends on how users want to handle the issue. I really like the newly written documentation and I believe that should answer a lot of questions.

Reply:
OK, let's merge as is and if people want more hints those can be added later.